packages/airqo-device-analysis/build/lib/airqloudanalysis/airqoairqloudanalysis/datainitialisation.py

```
@@ -0,0 +1,711 @@
+            # data starts after startdate and ends before enddate
+            elif startdate > existing_start and enddate < existing_end:
+                new_data_start = process_data(airqloud_df, existing_end, enddate)
+                new_data_end = process_data(airqloud_df, startdate, existing_start)
+                additional_data.extend([new_data_start, new_data_end])
+                updated_data = pd.concat([new_data_start, existing_data, new_data_end])
+
+            # data starts before startdate and ends after enddate
+            elif startdate < existing_start and enddate > existing_end:
+                pass
+
+            finaldf = pd.concat([existing_data] + additional_data) if additional_data else existing_data
```

Ruff (0.15.12)

[warning] Consider `[existing_data, *additional_data]` instead of concatenation

Replace with `[existing_data, *additional_data]`

(RUF005)